xml/installation-installation-designate-designate_migrate_zone_pool_to_worker_producer.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<section xml:id="DNS-MIGRATE"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>Migrate Zone/Pool to Worker/Producer after Upgrade</title>
+ <para>
+  To migrate the zone/pool scheme to the worker/producer scheme after upgrade
+  , follow these steps.
```

Comment:
I might change the above to something like `After upgrade, the following steps may be used to migrate the zone/pool scheme to a worker/producer scheme:`